src/uPlot.js

```diff
@@ -432,7 +438,13 @@ export default function uPlot(opts, data) {
 		return [round6(incrRoundUp(scaleMin, incr)), scaleMax];
 	}
 
-	// TODO: add ability to recompute & invalidate only specific scales. e.g. for series toggle
+	function getSerieMinMax(serie, data, _i0, _i1) {
+		if (serie.minMax == null) {
+			serie.minMax = getMinMax(data, _i0, _i1)
```

Comment:
to keep it uniform with scales structures let's keep the properties as separate. `s.min` and `s.max`

Reply:
It's done